hardware/electrical/power_supply.kicad_sch

```diff
@@ -5,6 +5,2413 @@
+    (pin "2" (uuid 8c147919-68f4-4693-8208-5d1609bff5d5))
+    (pin "3" (uuid c127d777-fcc3-4d35-a889-e7d6e9701667))
+    (pin "4" (uuid f0f39002-1152-467d-9c4d-78fcb38619cb))
+    (pin "5" (uuid e8d6ce2d-ecb2-4bf0-adcf-f3848c8ec979))
+    (pin "6" (uuid e5042b06-29ba-4cf6-905a-46febdfe7be1))
+  )
+
+  (symbol (lib_id "Connector:Conn_01x02_Female") (at 78.74 176.53 0) (unit 1)
+    (in_bom yes) (on_board yes) (fields_autoplaced)
+    (uuid 341c867b-81c3-46ea-966d-afade846817f)
+    (property "Reference" "J2" (id 0) (at 80.01 176.5299 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Value" "Conn_01x02_Female" (id 1) (at 80.01 179.0699 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Footprint" "Connector_PinSocket_2.00mm:PinSocket_1x02_P2.00mm_Vertical" (id 2) (at 78.74 176.53 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 78.74 176.53 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 84ee10c8-23b7-46ae-b3e4-2ece172778bd))
+    (pin "2" (uuid 98bb4ab5-96c1-4525-9efc-481b3daa9f95))
+  )
+
+  (symbol (lib_id "Device:R") (at 142.24 59.69 180) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 38d206ae-6005-4f27-9025-0773a3704a27)
+    (property "Reference" "R5" (id 0) (at 144.78 59.69 90))
+    (property "Value" "1k" (id 1) (at 139.7 59.69 90))
+    (property "Footprint" "Resistor_SMD:R_0805_2012Metric_Pad1.20x1.40mm_HandSolder" (id 2) (at 144.018 59.69 90)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 142.24 59.69 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid c7f7cd3b-ce3c-4fd0-b6ef-bba0d85ff3e5))
+    (pin "2" (uuid cedc703b-68c1-4e50-9558-84c987493f5a))
+  )
+
+  (symbol (lib_id "Transistor_FET:IRLML5203") (at 162.56 48.26 90) (mirror x) (unit 1)
+    (in_bom yes) (on_board yes) (fields_autoplaced)
+    (uuid 3f25646e-845d-4d3b-a015-d285ed685aa1)
+    (property "Reference" "Q3" (id 0) (at 162.56 55.88 90))
+    (property "Value" "IRLML5203" (id 1) (at 162.56 58.42 90)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Footprint" "Package_TO_SOT_SMD:SOT-23" (id 2) (at 164.465 53.34 0)
+      (effects (font (size 1.27 1.27) italic) (justify left) hide)
+    )
+    (property "Datasheet" "https://www.infineon.com/dgdl/irlml5203pbf.pdf?fileId=5546d462533600a40153566868da261d" (id 3) (at 162.56 48.26 0)
+      (effects (font (size 1.27 1.27)) (justify left) hide)
+    )
+    (pin "1" (uuid 77cba073-5968-4c51-a69b-1aabe357f3c6))
+    (pin "2" (uuid fbb20b14-fd19-4333-8acc-e38cba1ddb74))
+    (pin "3" (uuid 69ee7e1b-362f-458c-a3e0-54a30143927f))
+  )
+
+  (symbol (lib_id "Device:R") (at 49.53 63.5 0) (mirror x) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 42432c3c-6b80-4783-bc6f-de074e39d785)
+    (property "Reference" "R1" (id 0) (at 46.99 63.5 90))
+    (property "Value" "R" (id 1) (at 52.07 63.5 90))
```

Comment:
R1 (part of the battery protection circuit) doesn't seem to have a value

Reply:
Value of R1 limits the charging current which depends on the battery that we end up using. I suppose 2.5k ohm will do for now.

Reply:
The value can be TBD, I think this is standard for cases like this where it is dependent on other decisions being made or even if the value is determined from some measurement made _in situ_ on the final board.